tests/test_interface.rb

```diff
@@ -958,6 +959,44 @@ def test_interface_ipv4_arp_timeout
     assert_raises(RuntimeError) { nonvlanint.ipv4_arp_timeout = 300 }
   end
 
+  def test_interface_fabric_forwarding_anycast_gateway
+    # Setup
+    config('no interface vlan11')
+    int = Interface.new('vlan11')
+    config('feature fabric forwarding')
+    foo = VxlanGlobal.new
+    foo.anycast_gateway_mac = '1223.3445.5668'
+
+    # 1. Testing default
+    int.fabric_forwarding_anycast_gateway = false
+    assert_equal(int.default_fabric_forwarding_anycast_gateway,
```

Comment:
Eh? assert_equal is the correct thing to use here since you're comparing against the default value.

However, the line above should be:

```ruby
int.fabric_forwarding_anycast_gateway = int.default_fabric_forwarding_anycast_gateway
```